packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -265,7 +268,7 @@ export const LocationDetailViewTable = (): JSX.Element => {
       return (
         <TableRow key={index}>
           {LOCATION_DETAIL_VIEW_COLUMNS.map((column) => {
-            if (row.key === prefix) {
+            if (row.key === '') {
               // Don't render the current prefix as a row
```

Comment:
```suggestion
            // ignore empty strings
            if (row.key === '') {
```
              
But wonder if it makes sense to filter out the "base" folder in the result parsing?

Reply:
Yup, fixed this one.